tools/generator_python3.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python
+# coding=ascii
```

Comment:
Default encoding for Python 3 is now UTF-8. I'm pretty sure this is ignored or at the vary least confusing as your text editor is using UTF-8 as well.

Yes, I know that I had originally added these two lines back in 2015, but times have changed. These are no longer desired or required.